### PR TITLE
add test for user commands

### PR DESCRIPTION
**Pull request by karakayasemi**
## Description
Unit test for User commands

## Motivation and Context
"All code is guilty, until proven innocent."

## How Has This Been Tested?
Locally tested with sqlite and mysql

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by karakayasemi** [comment]
Drone failed in interactive test cases, I used commandTester's setInputs function for interactive dialog answers and it worked locally, I will investigate more.

**Comment by karakayasemi** [comment]
Travis tests are passing. As I understood our drone environment is not appropriate for interactive input. we need to add tty: true parameters to somewhere in docker configuration to solve the issue.

**Comment by karakayasemi** [comment]
I added the interactive test case problem as "TODO" and  commented out interactive test cases

File: tests/Core/Command/User/ListUserGroupsTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    /** @var CommandTester */
+    private $commandTester;
+
+    protected function setUp() {
+        parent::setUp();
+
+        $command = new ListUserGroups(\OC::$server->getUserManager(), \OC::$server->getGroupManager());
+        $this->commandTester = new CommandTester($command);
+        $user = $this->createUser('user1');
+        \OC::$server->getGroupManager()->createGroup('testgroup');
```
Comment (DeepDiver1975) [inline tests/Core/Command/User/ListUserGroupsTest.php]: looks like a GroupTrait would be handy ....

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by karakayasemi** [comment]
What about commented test cases? Should we open an issue about this topic?

**Comment by DeepDiver1975** [comment]
> What about commented test cases? Should we open an issue about this topic?

yes please

**Pull request merged**